distributed_resque_worker.gemspec

```diff
@@ -33,10 +33,10 @@ Gem::Specification.new do |spec|
   spec.require_paths = ['lib']
 
   spec.add_development_dependency 'bundler', '~> 1.17.3'
-  spec.add_development_dependency 'rake', '~> 10.0'
-  spec.add_development_dependency 'rspec', '~> 3.0'
+  spec.add_development_dependency 'rake', '~> 13.0.3'
+  spec.add_development_dependency 'rspec', '~> 3.9.0'
   spec.add_development_dependency 'rubocop', '~> 0.48'
-  spec.add_development_dependency 'webmock', '~> 3.4.2'
-  spec.add_dependency 'aws-sdk', '1.11.1'
+  spec.add_development_dependency 'webmock', '~> 3.12.2'
+  spec.add_dependency 'aws-sdk', '3.0.2'
```

Comment:
@TruptiHosmani did we not have to change any methods in aws_helper for this?
IIRC, AWS ruby SDK v3 was quite different from v1 no?

Reply:
No, I did not have to. Even I was surprised.